ProcessOptimizer/learning/gbrt.py

```diff
@@ -46,6 +46,11 @@ def __init__(self, quantiles=[0.16, 0.5, 0.84], base_estimator=None,
         self.base_estimator = base_estimator
         self.n_jobs = n_jobs
 
+    def __sklearn_tags__(self):
```

Comment:
Why isn't this automatic from RegressorMixin? https://github.com/scikit-learn/scikit-learn/blob/main/sklearn/base.py line 614 should make this fix not needed.

Reply:
We do something funky at lines  107-109 in learning/cook_estimator.py for the specific estimator, that caused the trouble.

![image](https://github.com/user-attachments/assets/ea043f62-75f8-473d-84e2-d10680581068)

Maybe we lose some inheritance somewhere along the way

Reply:
Switching the inheritance order works, see  #302